index.css

```diff
@@ -453,7 +468,3 @@ select.modal-question {
     }
 
 }
-
-.hidden {
-    display: none;
-}
```

Comment:
github prefers when there's an empty line at the end of each file (I think its bc of c++) but could you add that?